cognee/modules/data/extraction/knowledge_graph/extract_content_graph.py

```
@@ -2,12 +2,27 @@
 from pydantic import BaseModel
 from cognee.infrastructure.llm.get_llm_client import get_llm_client
 from cognee.infrastructure.llm.prompts import render_prompt
+from dotenv import load_dotenv
+import os
 
 
 async def extract_content_graph(content: str, response_model: Type[BaseModel]):
     llm_client = get_llm_client()
```

Ruff (0.8.2)

Local variable `llm_client` is assigned to but never used

Remove assignment to unused variable `llm_client`

(F841)